### ISSUE TITLE
[Feature-Resquest?] Now, with active development stopped, allow us to update the included 7-Zip standalone manually

### ISSUE THREAD

**Issue by NewRavenGer** [MikeMaximus/gbm#291]
Hey there,

now that you have stopped active development of GBM (thanks for your work, again!) it may be reasonable to let the users update the included 7za standalone version by themselves. It made sense for security to check for a modified file in the past, but with no regular updates to GBM being planned, what was probably intended to be a security feature could very well become a security risk in the future. This could be a "do it at your own risk, agree that GBM doesn't validate 7za.exe anymore" optional thing you have to enable first, of course. 

Michael

**Comment by MikeMaximus** [comment]
Both the 7z Custom Arguments and Custom Location were broken, so thanks for bringing this up. 😄  If you don't want to wait for the next release you can use the following build.

[GBM - 7z Custom Fix.zip](https://github.com/MikeMaximus/gbm/files/10272927/GBM.-.7z.Custom.Fix.zip)

This is backwards compatible with 1.3.3,  just replace your GBM.exe with the one in the archive.  This build also contains the DPI fix from #290.

**Comment by MikeMaximus** [comment]
I just want to add that I will continue to update the components with maintenance releases,  especially when I'm aware of a security issue.

**Comment by NewRavenGer** [comment]
I never tried the custom arguments, actually. And probably I should... I mean, my Skyrim backup is about 1,1 GB in size... I sure could save a few MB here and there. I used, however, the custom location since 7-Zip 22.01 was released... and while I didn't checked if it actually worked (yeah, not so smart...), that release was also where I tested if I could update the included version by myself. Which didn't work... and then I wrote this issue here and used the custom location... and here we are. Found a bug I didn't intended to find while suggesting a change that, if you plan to update the components in the future, isn't even required.
